src/service/ui/service.js

```diff
@@ -53,6 +53,7 @@ export const DeviceChooser = GObject.registerClass({
         super._init({
             use_header_bar: true,
             application: Gio.Application.get_default(),
+            title: params.title,
```

Comment:
`title` is already set in the template, as `Select a Device`, which is why `params.title` is set as the headerbar _subtitle_:

https://github.com/GSConnect/gnome-shell-extension-gsconnect/blob/3758807fac3f2b3c09a30a1767cf7f551d27d915/data/ui/service-device-chooser.ui#L15